emails/models.py

```diff
@@ -682,7 +682,10 @@ def full_address(self):
 def check_user_can_make_another_address(profile: Profile) -> None:
     if profile.is_flagged:
         raise AccountIsPausedException()
-    if profile.at_max_free_aliases and not profile.has_premium:
+    # MPP-3021: return early for premim users to avoid at_max_free_aliases DB query
```

Comment:
```suggestion
    # MPP-3021: return early for premium users to avoid at_max_free_aliases DB query
```
(compliment) I like this practice of adding the Jira issue number in the comments. While I think it makes it easier to understand code, I also think it makes the code messy. Let's revisit how and when we use the Jira issue on comments versus commits on [MPP-3216](https://mozilla-hub.atlassian.net/browse/MPP-3216).

Reply:
fixed typo.